.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
```

Comment:
Should we also add `pull_request` here? It doesn't seem to run in this repo because of 'push'?

Reply:
Actions run in the repo they see as their origin, so in the case of this PR my fork. Once merged they will run in this repo. Otherwise people could steal your secrets :)

Reply:
But there must a be a way to have pull requests also have them run in here, else no PR will ever run.. :-D

Reply:
All I know 💯 sure is that I use the same config for my projects and the PRs do get the actions executed!